README.md

```diff
@@ -415,45 +463,114 @@ This field SHOULD NOT be used to sign arbitrary data, such as signature challeng
+
+| Proof Caveats | Delegated Caveats | Is Valid? | Comment                              |
+|---------------|-------------------|-----------|--------------------------------------|
+| `[{}]`        | `[{}]`            | Yes       | Equal                                |
+| `[x]`         | `[x]`             | Yes       | Equal                                |
+| `[x]`         | `[{}]`            | No        | Escalation to any                    |
+| `[{}]`        | `[x]`             | Yes       | Introduces a new caveat              |
+| `[x, y]`      | `[x]`             | Yes       | Removes a capability                 |
+| `[x, y]`      | `[x, (y + z)]`    | Yes       | Attenuates existing caveat           |
+| `[x, y]`      | `[x, y, z]`       | No        | Escalation by adding new capability  |
+
+Note that for consistency in this syntax, the empty array MUST be equivalent to disallowing the capability. Conversely, an empty object MUST be treated as "no caveats".
 
-### 3.2.7 Proof of Delegation
+| Proof Caveats | Comment                                                       |
+|---------------|---------------------------------------------------------------|
+| `[]`          | No capabilities                                               |
+| `[{}]`        | Full capabilities for this resource/ability pair (no caveats) |
 
-Attenuations MUST be satisfied by matching the attenuated capability to a proof in the `prf` array ([§3.2.7.1](#3271-prf-field))
+### 3.2.6 Proof of Delegation
 
-Checked proofs MUST be resolvable by the recipient. A proof MAY be left unresolvable if it is not used as support for the top-level UCAN's capability chain. The exact format MUST be defined in the relevant transport specification. Some examples of possible formats include: a JSON object payload delivered with the UCAN, a federated HTTP endpoint, a DHT, or shared database.
+Attenuations MUST be satisfied by matching the attenuated capability to a proof in the [`prf` array][prf field].
 
-#### 3.2.7.1 `prf` Field
+Proofs MUST be resolvable by the recipient. A proof MAY be left unresolvable if it is not used as support for the top-level UCAN's capability chain. The exact format MUST be defined in the relevant transport specification. Some examples of possible formats include: a JSON object payload delivered with the UCAN, a federated HTTP endpoint, a DHT, or a shared database.
 
-The `prf` field MUST contain the content address ([§6.5](#65-content-identifiers)) of UCAN proofs (the "inputs" of a UCAN). 
+#### 3.2.6.1 `prf` Field
 
-#### 3.2.7.2 Examples
+The `prf` field MUST contain the [content address][content identifiers] of UCAN proofs (the "inputs" of a UCAN). 
```

Comment:
FWIW my initial reaction was that my vote would go out to the weird `{ "/": "<cid>" }` syntax for CIDs with dag-json.
The thinking was that we'd get e.g. `kubo pin` support and UCAN traversal via js-multiformats etc. for free, but that's not actually the case: The JWT encoding format with dotted and base64-ed payload + signature is in our way for that, so you'd need a canonicalization format like ucan-ipld in between anyways, so I'd say it's not worth the weirdness & extra bytes.